extensions/tags/js/src/forum/components/TagHero.js

```diff
@@ -7,7 +8,7 @@ export default class TagHero extends Component {
     const color = tag.color();
 
     return (
-      <header className={'Hero TagHero' + (color ? ' TagHero--colored' : '')} style={color ? { '--hero-bg': color } : ''}>
+      <header class={classList('Hero TagHero', { 'TagHero--colored': color })} style={color ? { '--hero-bg': color } : undefined}>
```

Comment:
why move away from `className` in certain places abut not others 🤔

Reply:
Honestly, just a case of where I could be bothered to change it...

Reply:
is there a reason we'd want to move towards just using `class`?

Reply:
Smaller bundle size? 😁